MAUI/ImageEditor/Crop.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: Crop in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about crop support in Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Crop in .NET MAUI ImageEditor (SfImageEditor)
+
+Using the Image Editor's cropping tool, you can select and crop a particular section of an image.
```

Comment:
Using the Image Editor's cropping tool, you can select and crop a section of an image.